README.md

```diff
@@ -24,7 +24,7 @@ The module has been tested with:
 
 ## Disk Support for Materialize on GCP
 
-This module supports configuring disk support for Materialize using local SSDs in GCP with OpenEBS and lgalloc.
+This module supports configuring disk support for Materialize using local SSDs in GCP with OpenEBS.
```

Comment:
We don't use OpenEBS anymore. It's still installed until part2, since we need to clean up the volumes with this version first.